### ISSUE TITLE
NVDA does not work with pdf preview

### ISSUE THREAD

**Issue by richardMSFT** [microsoft/AzureStorageExplorer#6789]
### Preflight Checklist

- [X] I have installed the [latest version of Storage Explorer](https://github.com/Microsoft/AzureStorageExplorer/releases/latest).
- [X] I have checked existing resources, including the [troubleshooting guide](https://docs.microsoft.com/azure/storage/common/storage-explorer-troubleshooting) and the [release notes](https://github.com/microsoft/AzureStorageExplorer/releases).
- [X] I have searched for [similar issues](https://github.com/Microsoft/AzureStorageExplorer/issues).

### Storage Explorer Version

1.29.0-dev

### Architecture

i86

### Storage Explorer Build Number

20230417.5

### Platform

All

### Bug Description

Edge (built on Chromium) works with NVDA screen reader, but doesn't for pdf previewer.

### Steps to Reproduce

Use NVDA to read contents for pdf preview

### Actual Experience

NVDA not reading contents of pdf preview

### Additional Context

Check if other screen readers are working with pdf preview
